DashboardEmbedding/README.md

```diff
@@ -0,0 +1,91 @@
+# Embed Dashboards with SSO using Token
+
+This sample demonstrates the embedding the dashboard with iFrame based Single Sign-on(SSO) using token based authentication.
+
+## Dashboard view
+
+![Dashboard View](https://github.com/boldbi/blazor-webassembly-sample/assets/91586758/9719d6e4-f0e0-4254-afc6-051489550eb6)
+
+ #### Help link
+
+ * https://help.boldbi.com/embedded-bi/faq/where-can-i-find-the-product-version/
+
+ #### Supported browsers
+  
+  * Google Chrome, Microsoft Edge, Mozilla Firefox.
+
+ ## Configuration
+
+  1. Requires the following properties to be configured as mandatory:
```

Comment:
Follow the order in the table as you mentioned here.

Reply:
Table order using properties order changed.